src/test/java/com/wikia/webdriver/testcases/discussions/PostsList.java

```diff
@@ -55,6 +55,12 @@ public void anonUserOnDesktopCanSeePostsList() {
 
   @Test
   @Execute(browserSize = DESKTOP_RESOLUTION, asUser = User.ANONYMOUS)
+  public void anonCanSwitchBetweenLatestAndTrendingTabOnDesktop() {
```

Comment:
please follow naming pattern in this file and rename `anonCanSwitchBetweenLatestAndTrendingTabOnDesktop` --> `anonUserOnDesktopCanSwitchBetweenLatestAndTrendingTab`